api/RadioNodeList.json

```diff
@@ -6,40 +6,40 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#radionodelist",
         "support": {
           "chrome": {
-            "version_added": "34"
+            "version_added": "1"
```

Comment:
Since this interface has `value` as the only member, I don't think it's meaningful to say that it's supported before the `value` property. The custom test for this will return true regardless of what kind of object `form.elements.namedItem('radio')` returns. In particular if it's a `NodeList` I don't think that should count as supported.